automation/devops_automation_infra/utils/node_daemon.py

```diff
@@ -0,0 +1,59 @@
+import json
+import logging
+import os
+
+from automation_infra.utils import waiter, timer
+from devops_automation_infra.utils import consul
+from infra.model import cluster_plugins
+from pipeng.services import pipe
+FACE_FLOW_TYPE = 1
+BODY_FLOW_TYPE = 2
+SERVICE_ADDRESS_SEPARATOR = ":"
+OPEN_VINO_CONSUL_PATH = "supervisor_local-agent-default/pipe_defs:OV"
+TRT_CONSUL_PATH = "supervisor_local-agent-default/pipe_defs:TRT"
+THIS_DIRNAME = os.path.dirname(os.path.realpath(__file__))
+ZMQ_KEYS_DIR = os.path.realpath("%s/../../../zmq/certificates" % THIS_DIRNAME)
+from pipeng.utils import tracks
+from devops_automation_infra.k8s_plugins.consul import Consul
+from devops_automation_infra.k8s_plugins.kafka import Kafka
```

Comment:
All the `from ...` imports should be together

In general it should look something like this:

`
import ..
import

from ...
from ... 
from ...

CONST
CONST


def ...
`